repo/packages/S/spark/12/config.json

```diff
@@ -156,6 +156,67 @@
+                        },
+                        "opt_mesosphere_dir": {
+                            "default": "/opt/mesosphere",
+                            "description": "Volume to mount in docker for modules.",
+                            "type": "string"
+                        },
+                        "dcos_service_account_credential_dir": {
+                            "default": "/run/dcos/etc/mesos",
+                            "description": "Volume to mount in docker for service-account credential.",
+                            "type": "string"
+                        },
+                        "module_json": {
+                            "default": "file:///opt/mesosphere/etc/mesos-scheduler-modules/dcos_authenticatee_module.json",
+                            "description": "JSON file describing the module within the module_dir.",
+                            "type": "string"
+                        },
+                        "module_name": {
+                            "default": "com_mesosphere_dcos_ClassicRPCAuthenticatee",
+                            "description": "Authenticatee module name.",
+                            "type": "string"
+                        },
+                        "dcos_service_account_credential_json": {
+                            "default": "file:///run/dcos/etc/mesos/agent_service_account.json",
+                            "description": "File containing DCOS service-account credential.",
+                            "type": "string"
+                        },
+                        "mesos_credential": {
+                            "default": "{\"principal\": \"mesos_agent\", \"secret\": \"dcos_spark\"}",
+                            "description": "Mesos credentials. The secret is not used.",
+                            "type": "string"
+                        },
+                        "ssl_files_dir": {
+                            "default": "/run/dcos/pki",
+                            "description": "Directory containing SSL-related files.",
+                            "type": "string"
+                         },
+                        "ssl_cert_file": {
+                            "default": "/run/dcos/pki/tls/certs/mesos-slave.crt",
+                            "description": "Path to SSL cert file on the host.",
+                            "type": "string"
+                         },
+                         "ssl_key_file": {
+                              "default": "/run/dcos/pki/tls/private/mesos-slave.key",
+                              "description": "Path to SSL key file on the host.",
+                              "type": "string"
+                         },
+                         "ssl_ca_dir": {
+                              "default": "/run/dcos/pki/certs",
+                              "description": "Path to SSL CA directory on the host.",
+                              "type": "string"
+                         }
```

Comment:
https://github.com/mesosphere/universe/pull/524

On Tue, Jun 28, 2016 at 10:27 AM, Adam <notifications@github.com> wrote:

> In repo/packages/S/spark/12/config.json
> <https://github.com/mesosphere/universe/pull/497#discussion_r68803083>:
>
> > +                         },
> > +                        "ssl_cert_file": {
> > +                            "default": "/run/dcos/pki/tls/certs/mesos-slave.crt",
> > +                            "description": "Path to SSL cert file on the host.",
> > +                            "type": "string"
> > +                         },
> > +                         "ssl_key_file": {
> > +                              "default": "/run/dcos/pki/tls/private/mesos-slave.key",
> > +                              "description": "Path to SSL key file on the host.",
> > +                              "type": "string"
> > +                         },
> > +                         "ssl_ca_dir": {
> > +                              "default": "/run/dcos/pki/certs",
> > +                              "description": "Path to SSL CA directory on the host.",
> > +                              "type": "string"
> > +                         }
>
> Yes, we need to make comments. Please create a PR.
> Since this needs to go into Universe 3.x for Argus, can you please make
> the PR against that repo?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mesosphere/universe/pull/497/files/d3b313295487cbe6351e64a7006589b081f4fe62#r68803083>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAJYhqy0WPXuxRp4FmpL6ulvS8bQJZpIks5qQVmUgaJpZM4I6Xgy>
> .
>



-- 
Michael Gummelt
Software Engineer
Mesosphere